careers/base/templates/base.html

```diff
@@ -10,6 +10,7 @@
     <title>{% block page_title %}mozilla.org{% endblock %}</title>
 
     {# css #}
+    <link href='http://fonts.googleapis.com/css?family=Fira+Sans' rel='stylesheet' type='text/css'>
```

Comment:
So upon further reflection, I don't think we want to load an entirely new font just for a single word; how about we just use Open-Sans for the logo since it's already loaded on the page? I checked and it looks acceptably fine.